.gitignore

```diff
@@ -4,3 +4,4 @@ build/dev
 docs/*
 !docs/*.conf.json
 !docs/*.ico
+.idea/*
```

Comment:
Yeah but I have projects where I want to commit some files from the .idea directory.
But if you don't want it in your gitignore I undo it in my branch.

And your gitignore doesn't have a newline at the end either?